pages/dashboard/developer.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <DashboardPage>
+    <div class="section-header columns">
+      <h3 class="column-grow-1">Developer Tools</h3>
+    </div>
+    <modal v-show="main_component">
+      <section class="essentials">
+        <h3>Authorization Token</h3>
+        <label>
+          <span>
+            Your authorization token can be used with the Modrinth API and for
+            the Minotaur Gradle plugin. However, it must be kept secret!
+          </span>
+          <button @click="copyToken">Copy to clipboard.</button>
+        </label>
+        <h3>Revoke your token</h3>
+        <label>
+          <span
+            >Beware, this will log you out of Modrinth, and you will have to
+            login again to access modrinth with a new token.</span
+          >
+          <button @click="showDelPopup">Revoke Token</button>
+        </label>
+      </section>
+    </modal>
+    <modal v-show="revoke_component">
+      <RevokeTokenHint @wantClose="showDelPopup"></RevokeTokenHint>
+    </modal>
```

Comment:
Yeah, should be just moving this and all related code to the `/dashboard/settings`